include/sys/arc_impl.h

```diff
@@ -176,6 +177,183 @@ typedef struct l1arc_buf_hdr {
 	abd_t			*b_pabd;
 } l1arc_buf_hdr_t;
 
+typedef enum l2arc_dev_hdr_flags_t {
+	L2ARC_DEV_HDR_EVICT_FIRST = (1 << 0)	/* mirror of l2ad_first */
+} l2arc_dev_hdr_flags_t;
+
+/*
+ * Pointer used in persistent L2ARC (for pointing to log blocks & ARC buffers).
```

Comment:
What does `ARC buffers` mean here?  I think that this (l2arc_log_blkptr_t) is used for pointing to `l2arc_log_blk_phys_t` only, and `l2arc_log_ent_phys_t` is used for describing an individual ARC header/buffer.

Reply:
Of course you are correct. Will remove.